dev/com.ibm.ws.persistence.ol_fat/test-applications/consumer/src/com/ibm/ws/persistence/consumer/web/AbstractServlet.java

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.persistence.consumer.web;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@SuppressWarnings("serial")
+public class AbstractServlet extends HttpServlet {
```

Comment:
this class is doing the same thing as the `FATServlet` class provided by the `componenttest-1.0` feature.  Lets remove this class and add `componenttest-1.0` to the server.xml and have the other test servlets extend `FATServlet` instead of this